tracing/config/src/main/java/io/helidon/tracing/config/TracingConfig.java

```diff
@@ -0,0 +1,227 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.tracing.config;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import io.helidon.common.OptionalHelper;
+import io.helidon.config.Config;
+
+/**
+ * Configuration of traced environment.
```

Comment:
Since the "Env" prefix was removed, should eliminate uses of the term "environment" to avoid confusion. Perhaps just "Tracing configuration" or perhaps include a description of the scope, e.g. "global", "application" etc.